src/js/pensions/helpers.jsx

```diff
@@ -148,7 +147,7 @@ export const uploadMessage = (
 
 export const aidAttendanceEvidence = (
   <div className="usa-alert usa-alert-info no-background-image">
-    <strong>If you’re claiming non-service-connected pension benefits with Aid and Attendance</strong>, the evidence must show that you:
+    <strong>If you’re claiming non-service-connected pension with Aid and Attendance benefits</strong>, your supporting documents must show that you:
```

Comment:
Should the opening <strong> tag be moved to just before non-service to align with the style on line 159?